client/src/styles/Card.scss

```diff
@@ -0,0 +1,63 @@
+@import "_vars.scss";
```

Comment:
my attempt to generalize the card component

Reply:
love this, so much cleaner